ci/workflows/main-cron.yml

```diff
@@ -21,7 +21,7 @@ auto-retry: &auto-retry
 
 steps:
   - label: "build"
-    command: "ci/scripts/build.sh -t ci-release -p ci-release"
+    command: "ci/scripts/build.sh -t ci-dev -p ci-dev"
```

Comment:
🤔👀 Why change this to `dev`?

Reply:
Previously it is dev. I changed it to release expecting the data ingestion phase will be short, but the building time becomes longer, and the total time is almost the same. So I change it back to dev

Reply:
Test again, release build is more promising.  https://buildkite.com/risingwavelabs/main-cron/builds/196